### PR TITLE
Add a prompt to remember writing migrations

### PR DESCRIPTION
**Pull request by ewjoachim**
Also, encourage people not to discard the list completely.

### Successful PR Checklist:
- [ ] Tests < not applicable
- [x] Documentation (optionally: [run spell checking](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#build-the-documentation))
- [x] Had a good time contributing? (if not, feel free to give some feedback)

### CODE REVIEW

**Comment by ewjoachim** [comment]
Hmm, we can do better !

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Closes #<ticket number>
 
+<!-- Please do not remove this, even if you think you don't need it -->
 ### Successful PR Checklist:
 - [ ] Tests
 - [ ] Documentation (optionally: [run spell checking](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#build-the-documentation))
+- [ ] Migration, if applicable
```
Comment (CorBott) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
- [ ] Migration procedure, if applicable
```

Comment (ewjoachim) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I've changed it to "SQL Migrations". This refers to [this part](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#migrations) in the contributing doc.

**Comment by k4nar** [comment]
I think that editing might be a bit confusing for some contributors. It's the last step in the process, so it shouldn't be confusing :) .

Maybe instead we could add a CI check with an error if you touch the schema but not the migrations? Or if you touch a migration corresponding to an earlier version.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Closes #<ticket number>
 
+<!-- Please do not remove this, even if you think you don't need it -->
 ### Successful PR Checklist:
+<!-- In case of doubt, we're here to help. CONTIRBUTING.md might help too -->
 - [ ] Tests
-- [ ] Documentation (optionally: [run spell checking](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#build-the-documentation))
-- [ ] Had a good time contributing? (if not, feel free to give some feedback)
+  - ([ ] not applicable ?)
```
Comment (elemoine) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: (nit) no space before question mark

**Comment by ewjoachim** [comment]
> I think that editing might be a bit confusing for some contributors. It's the last step in the process, so it shouldn't be confusing :) .
Maybe instead we could add a CI check with an error if you touch the schema but not the migrations? Or if you touch a migration corresponding to an earlier version.

Ok so I I understood correctly, the idea is that once we'll have the pum checks, we won't need this checkbox, and I agree. Let's check with pum !

**Pull request merged**